Firestore/core/src/firebase/firestore/remote/exponential_backoff.h

```diff
@@ -27,6 +27,19 @@ namespace firebase {
 namespace firestore {
 namespace remote {
 
+using Milliseconds = util::AsyncQueue::Milliseconds;
+
+/**
+ * Initial backoff time in milliseconds after an error. Set to 1s according to
+ * https://cloud.google.com/apis/design/errors.
+ */
+const Milliseconds kDefaultBackoffInitialDelay = Milliseconds(1000);
+
+const double kDefaultBackoffFactor = 1.5;
+
+/** Maximum backoff time in milliseconds. */
+const Milliseconds kDefaultBackoffMaxDelay = Milliseconds(60 * 1000);
```

Comment:
Similar to my feedback elsewhere, I think these could be moved to the `.cc` file and made `static` ... unless we intentionally want to make them visible to other components (e.g. in case a component wants to initialize backoff with a custom value for one parameter but use the defaults for the others or something).  I guess we _did_ make these `public` members on Android (rather than `private`) though, so I guess we could go either way.  No strong feelings.

Reply:
I've gone with leaving them as `static const`s in the `.h` file, but open to other suggestions // tradeoffs.